docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
+
+If you download it from a HTTP server then the file per default is not executable, we need to change it with `chmod`. After that is done we can run it, it's a self extracting shell script.
```

Comment:
* Same thing here: New sentence starts after 'executable'.
* Also after "run it" but there I think we should instead make it the same sentence: ", it's" -> "since it's".